api/incubator/src/main/java/io/opentelemetry/api/incubator/events/EventBuilder.java

```diff
@@ -5,28 +5,107 @@
 
 package io.opentelemetry.api.incubator.events;
 
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.incubator.logs.AnyValue;
+import io.opentelemetry.api.logs.Severity;
+import io.opentelemetry.context.Context;
 import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
 import java.util.concurrent.TimeUnit;
 
 /** The EventBuilder is used to {@link #emit()} events. */
 public interface EventBuilder {
 
+  /** Put the given {@code key} and {@code value} in the payload. */
+  default EventBuilder put(String key, String value) {
+    return put(key, AnyValue.of(value));
+  }
+
+  /** Put the given {@code key} and {@code value} in the payload. */
+  default EventBuilder put(String key, long value) {
+    return put(key, AnyValue.of(value));
+  }
+
+  /** Put the given {@code key} and {@code value} in the payload. */
+  default EventBuilder put(String key, double value) {
+    return put(key, AnyValue.of(value));
+  }
+
+  /** Put the given {@code key} and {@code value} in the payload. */
+  default EventBuilder put(String key, boolean value) {
+    return put(key, AnyValue.of(value));
+  }
+
+  /** Put the given {@code key} and {@code value} in the payload. */
+  default EventBuilder put(String key, String... value) {
+    List<AnyValue<?>> values = new ArrayList<>(value.length);
+    for (String val : value) {
+      values.add(AnyValue.of(val));
+    }
+    return put(key, AnyValue.of(values));
+  }
+
+  /** Put the given {@code key} and {@code value} in the payload. */
+  default EventBuilder put(String key, long... value) {
+    List<AnyValue<?>> values = new ArrayList<>(value.length);
+    for (long val : value) {
+      values.add(AnyValue.of(val));
+    }
+    return put(key, AnyValue.of(values));
```

Comment:
What do you think about the streaming equivalents for these?

```
List<AnyValue<?>> values = LongStream.of(value)
    .boxed()
    .map(AnyValue::of)
    .collect(Collectors.toList());
return put(key, AnyValue.of(values));
```

Reply:
we've generally avoided streams on the hotpath in Java instrumentation repo due to performance / allocation concerns

Reply:
Also, I tried to do this but got blocked by animal sniffer android compatibility 😢

Reply:
Ok, I'm Homer backing into the bushes. :)